hw2/04-background.js

```diff
@@ -1 +1,57 @@
-// Add your code here
+const button = document.querySelector("#start"); // start button click
+button.addEventListener("click", handleClick);
+
+const userInput = document.querySelector("#num"); // num seconds
+
+// Setup
+let intervalID;
+
+// Create an array of colors to select from
+const colors = [
+  "lightcoral",
+  "cadetblue",
+  "darkolivegreen",
+  "darkmagenta",
+  "darkgoldenrod",
+  "darkslateblue",
+  "cornflowerblue",
+];
```

Comment:
Love the colors you chose here!

Reply:
Thanks!  It's hard enough to look at a constant changing background through development.   I wanted to at least have colors that weren't irritating lol.